src/object/schema_manager.c

```diff
@@ -3032,6 +3117,144 @@ sm_is_system_class (MOP op)
   return sm_get_class_flag (op, SM_CLASSFLAG_SYSTEM);
 }
 
+/*
+ * sm_check_system_class_by_name () - Checks whether the class name is
+ *    the same as the system class name.
+ * return: int
+ * name(in): class simple name
+ */
+int
```

Comment:
```suggestion
bool
```
this function looks return true or false

Reply:
Okay, I'll change it.